tests/unit/serialize_test.py

```diff
@@ -0,0 +1,191 @@
+from rflx.expression import Variable
```

Comment:
All other existing unit tests are structured in the same way as the actual code. I'm not sure if its good idea to add unit tests with a feature-wise structuring in parallel. In the past we had similar feature-wise test modules like `verification_test` and `test_z3`. But in both cases some tests which should actually have been part of these test modules, where added to other test modules. Some tests were even duplicated. That's why I would recommend moving these test methods to the respective existing test modules. It should still be possible to only execute all serialize-related test methods using pytest`s `-k` option or alternatively by introducing [pytest makers](https://docs.pytest.org/en/stable/mark.html).